WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.java

```diff
@@ -126,12 +126,12 @@ public void onClick(View view) {
         }
     }
 
-    private void signoutWithConfirmation() {
+    private void signOutWordPressComWithConfirmation() {
         new AlertDialog.Builder(getActivity())
```

Comment:
This confirmation should be different (or there should be an additional confirmation) if signing out will remove local drafts.